### PR TITLE
Moved Downloads from New Tab page to Control Center #432

### LINKED ISSUE #432: Move Downloads from New Tab page to Control Center
Downloads occupy quite prominent spot on the new tab page but our supposition is that they are not that commonly used.
Lets move the download under Control Center menu (below Home) and open it on a dismissible Card (with close button).

### PR DESCRIPTION
**Pull request by shahenAnt**
Fix #432 

## Implementation details
Moved Downloads from home screen to Control Center.

## Checklist:

- [x] My squashed commit messages follows the [7 golden rules](https://chris.beams.io/posts/git-commit/)
- [x] I updated or created necessary unit tests
- [x] I've tested my changes in Dark Mode, iPad, iOS <= 12, Landscape and in German

### CODE REVIEW

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -1484,27 +1520,26 @@ extension BrowserViewController: TabDelegate {
...
-            if buttonPressed {
-                self.tabManager.selectTab(tab)
-            }
-        })
-        self.show(toast: toast)
+        self.openURLInNewTab(url: url, isPrivate: isPrivate)
     }
 }
 
+extension BrowserViewController: DownloadsDelegate {
```
Comment (chrmod) [inline Client/Frontend/Browser/BrowserViewController.swift]: Can we move this to separate file?

File: Client/Frontend/Library/DownloadsViewController.swift [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+import UIKit
+import Storage
+import Shared
+
+protocol DownloadsDelegate: AnyObject {
+    func downloadsDidRequestToOpenInNewTab(_ url: URL, isPrivate: Bool)
+    func downloads(didSelectURL url: URL, visitType: VisitType)
+}
+
+class DownloadsViewController: UIViewController {
```
Comment (chrmod) [inline Client/Frontend/Library/DownloadsViewController.swift]: Please put new files under UserAgent group.

**Comment by shahenAnt** [comment]
Ok. I haven’t thought about folder structure before. I will fix it next Monday.

**Comment by shahenAnt** [comment]
@chrmod I did according changes. Could you please review again.

**Pull request merged**